File: backend/services/enrollment-service/src/main/java/com/lmspilot/enrollment/EnrollmentServiceApplication.java
package com.lmspilot.enrollment;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class EnrollmentServiceApplication {
    public static void main(String[] args) { SpringApplication.run(EnrollmentServiceApplication.class, args); }
}

File: backend/services/enrollment-service/src/main/kotlin/com/lmspilot/enrollment/EnrollmentServiceApplication.kt
package com.lmspilot.enrollment

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class EnrollmentServiceApplication

fun main(args: Array<String>) {
    runApplication<EnrollmentServiceApplication>(*args)
}
